src/Sylius/Bundle/ApiBundle/Tests/Doctrine/ORM/QueryExtension/Shop/Product/EnabledVariantsExtensionTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Tests\Doctrine\ORM\QueryExtension\Shop\Product;
+
+use ApiPlatform\Doctrine\Orm\Extension\QueryCollectionExtensionInterface;
+use ApiPlatform\Doctrine\Orm\Extension\QueryItemExtensionInterface;
+use ApiPlatform\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\HttpOperation;
+use Doctrine\ORM\QueryBuilder;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\ApiBundle\Doctrine\ORM\QueryExtension\Shop\Product\EnabledVariantsExtension;
+use Sylius\Bundle\ApiBundle\SectionResolver\AdminApiSection;
+use Sylius\Bundle\ApiBundle\SectionResolver\ShopApiSection;
+use Sylius\Bundle\CoreBundle\SectionResolver\SectionProviderInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+final class EnabledVariantsExtensionTest extends TestCase
+{
+    private EnabledVariantsExtension $extension;
+
+    /** @var SectionProviderInterface|MockObject */
+    private $sectionProvider;
+
+    protected function setUp(): void
+    {
+        $this->sectionProvider = $this->createMock(SectionProviderInterface::class);
+        $this->extension = new EnabledVariantsExtension($this->sectionProvider);
+    }
+
+    public function test_it_is_a_query_extension(): void
```

Comment:
With the newly added units we used the camelCase for the method names, so we are a little inconsistent